src/Umbraco.Infrastructure/BackgroundJobs/Jobs/WebhookFiring.cs

```diff
@@ -90,20 +89,29 @@ public async Task RunJobAsync()
 
     private async Task<HttpResponseMessage?> SendRequestAsync(IWebhook webhook, string eventName, string? serializedObject, int retryCount, CancellationToken cancellationToken)
     {
-        using var httpClient = new HttpClient();
+        using HttpClient httpClient = _httpClientFactory.CreateClient(Constants.HttpClients.WebhookFiring);
 
-        var stringContent = new StringContent(serializedObject ?? string.Empty, Encoding.UTF8, MediaTypeNames.Application.Json);
-        stringContent.Headers.TryAddWithoutValidation("Umb-Webhook-Event", eventName);
+        var request = new HttpRequestMessage(HttpMethod.Post, webhook.Url)
+        {
+            Version = httpClient.DefaultRequestVersion, VersionPolicy = httpClient.DefaultVersionPolicy,
+        };
+
+        // Add headers
+        request.Headers.TryAddWithoutValidation("Umb-Webhook-Event", eventName);
```

Comment:
I'd recommend adding the headers using `Add()` within the try/catch, so we ensure only valid request headers are added. We don't want users to add content headers like `Content-Type` (that is already set by the `StringContent`) or response headers (like `Last-Modified`)...

Reply:
Done 🚀